### DISCUSSION TITLE
V2.15-x86 for Win blocked and giving adverse virustotal result

### DISCUSSION THREAD

**Discussion by Eternal9100** [DNSCrypt/dnscrypt-proxy#2500] [Potential issues]
The file at:

https://github.com/DNSCrypt/dnscrypt-proxy/releases/download/2.1.5/dnscrypt-proxy-win32-2.1.5.zip

is giving me  malware detection on Virustotal. Chrome and Mozilla based browsers require override before download. The VT IS is:

99176d9ac53217976423b727f3b238c418b9e94cfeb55a9b9439fd01354f7535

at:

https://www.virustotal.com/gui/file/99176d9ac53217976423b727f3b238c418b9e94cfeb55a9b9439fd01354f7535

Is this a known problem? Please note that this only applies to 32 bit Win. x86-64 gives a clean scan. v 2.1.4 for 32 bit Win also gives a clean scan.

**Comment by lifenjoiner** [comment]
I'm not saying it is totally insane.
I would say it is false positive. Maybe that is because of `dnscrypt-proxy` can install itself as a service.

**Reply by Eternal9100** [reply]
Please see my amendment to the OP. v2.1.5 64 bit does not have this problem and v2.1.4 32 bit does not either. The installation as a service is by the same method. Pretty well any windows executable can be installed as a service though it is often extrinsic e.g. via NSSM.

**Reply by lifenjoiner** [reply]
`dnscrypt-proxy` is fully open source. If you don't trust the pre-built binary, just compile it yourself. That's easy.
Based on this, I really don't care what those machine generated result personally.

If you really care about those results, report a false positive to those vendors.

**Comment by sryze** [comment]
2.1.5 x64 build is also detected as malware by 3 antiviruses on VirusTotal:
https://www.virustotal.com/gui/file/12a45c1a215620c56f04531369e4ec733435b27eec2e1fd57c62ca22e491f393

**Reply by sryze** [reply]
It appears to be a common issue with Go-compiled binaries:
https://go.dev/doc/faq#virus
In fact I've just had Windows Defender complain about a trojan in a Hello World program compiled with Go 1.22.
